src/main/scala/com/lightbend/conductr/sbt/BundlePlugin.scala

```diff
@@ -395,93 +424,81 @@ object BundlePlugin extends AutoPlugin {
   }
 
   private def getConfig(config: Configuration, forAllSettings: Boolean): Def.Initialize[Task[String]] = Def.task {
-    val checkComponents = (checksConfigName in config).value match {
-      case (value, configName) if (forAllSettings || configName.isDefined) && value.nonEmpty =>
-        val checkInitialDelayValue = (checkInitialDelay in config).value
-        val checkInitialDelayInSeconds =
-          if (checkInitialDelayValue.toMillis % 1000 > 0)
-            checkInitialDelayValue.toSeconds + 1 // always round up
-          else
-            checkInitialDelayValue.toSeconds
-        Seq(
-          value.map(uri => s""""$uri"""").mkString(
-            s"""components = {
-                |  ${(normalizedName in config).value}-status = {
-                |    description      = "Status check for the bundle component"
-                |    file-system-type = "universal"
-                |    start-command    = ["check", "--initial-delay", "$checkInitialDelayInSeconds", """.stripMargin,
-            ", ",
-            s"""]
-                |    endpoints        = {}
-                |  }
-                |}""".stripMargin
-          )
-        )
-      case _ =>
-        Seq.empty[String]
-    }
-
-    def formatValue[T](format: String, valueAndConfigName: (T, Option[String])): Seq[String] =
+    def hasValue[T](valueAndConfigName: (T, Option[String])): Boolean =
       valueAndConfigName match {
-        case (value, configName) if forAllSettings || configName.isDefined => Seq(format.format(value))
-        case _                                                             => Seq.empty[String]
+        case (value, configName) if forAllSettings || configName.isDefined => true
+        case _                                                             => false
```

Comment:
I'd refactor this method as:

```
def hasValue[T](valueAndConfigName: (T, Option[String])): Boolean =
  forAllSettings || valueAndConfigName._2.isDefined
```

I should also say that to me is not clear what the implementation does. Based on the method's name (`hasValue`), I'd assume that the implementation should answer the question "does this pair have a value?". But, when I look at the implementation, it doesn't even look at the value, so I'm left wondering what this method is actually for. Sorry for the rant :)